frontend/src/components/pages/AdminTablePage.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+import { Spinner, Text, VStack, Image, Box } from "@chakra-ui/react";
+import AdminUserTable from "../common/AdminUserTable";
+import { AdminUser } from "../../types/UserTypes";
+
+import WarningIcon from "../../assets/warning-icon.png";
```

Comment:
I think we use the `eva` icon pack in our codebase and design system! Would you be able to import the correct icon from that library into the `icons.tsx` file and use that here?